frontend/src/components/housing/enquiry/EnquiryForm.vue

```diff
@@ -327,4 +351,13 @@ onMounted(async () => {
       </div>
     </div>
   </Form>
+  <ATSUserLinkModal
+    v-model:visible="atsUserLinkModalVisible"
+    @ats-user-link:link="
+      (atsUser: ATSUser) => {
+        formRef?.setFieldValue('atsClientNumber', atsUser.atsClientNumber);
```

Comment:
Please add `v-slot="{ setFieldValue }"` to the `<Form>` and remove `formRef?.`. The formRef should only be used inside of `<script>`.